app/models/visit.rb

```diff
@@ -12,7 +12,10 @@ class Visit
   attribute :vo_number, String
 
   validates_presence_of :visit_id
-  validates_size_of :slots, within: 1..MAX_SLOTS, on: :date_and_time, message: 'must be at least one and at most three'
+  validates_size_of :slots,
+    within: 1..MAX_SLOTS,
+    on: :date_and_time,
+    message: 'must be at least one and at most three'
```

Comment:
I have three problems with the existing code, which would be nice to fix while we're here.

1. It's strangely named (`size_of` is an alias for `length_of`, and `within` for `in`).
2. It uses the old style `validates_bar :foo` instead of `validate :foo, bar: …`.
3. I can't work out what `on: :date_and_time` does! Does it do anything?

```ruby
validate :slots, length: { in: 1..MAX_SLOTS }
```

Is probably enough, with the message moved to `en.yml`. We then get too short and too long for free, instead of this vague message:

```yaml
en:
  activemodel:
    errors:
      models:
        visit:
          attributes:
            slots:
              too_short: 'must be at least one'
              too_long: 'must be at most three'
```